content/doc/developer/handling-requests/deprecated-reflective-access.adoc

```diff
@@ -0,0 +1,18 @@
+---
+title: Deprecated reflective access
+layout: developersection
+---
+
+Before Jenkins 2.272 Stapler would access all fields ignoring visibility restrictions.
+
+Stapler now tries to access a field first and only if it gets access denied then will it force access via 
+javadoc:java.lang.reflect.AccessibleObject#setAccessible-boolean-[setAccessible(true)].
+
+This behaviour is not recommended anymore, and will trigger IllegalReflectiveAccess warnings when running Jenkins on Java 11.
```

Comment:
```suggestion
This behavior is not recommended anymore, and will trigger `IllegalReflectiveAccess` warnings when running Jenkins on Java 11.
```

I think we use Simplified English on the site, unsure.

Reply:
we do yes, I missed it on this one, (not used to spelling like that and my editor isn't configured for it)